contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -284,7 +308,7 @@ NXmpes(NXobject):
               exists: recommended
             identifier:
               exists: recommended
-          raw_data(NXdata):
+          raw_data(NXdata_mpes_detector):
```

Comment:
I thought we wanted to continue using just `NXdata`. Or do I remember it incorrectly?

Reply:
I think we agreed that we want to use these specialized base classes: NXdata_mpes, NXdata_mpes_detector, NXprocess_mpes. With the idea that eventually they should be extending NXdata or NXprocess, but for now we just copied everything from the original NXdata/NXprocess and added the fields that we want to have specific for MPES.

Reply:
If we go with NXdata here, we will again have the very long list of explicitly spelled out axisnames in the app def.